include/rnp/rnp.h

```diff
@@ -417,6 +430,96 @@ RNP_API rnp_result_t rnp_supports_feature(const char *type, const char *name, bo
  */
 RNP_API rnp_result_t rnp_supported_features(const char *type, char **result);
 
+/**
+ * @brief Add new security rule to the FFI. Security rules allows to override default algorithm
+ *        security settings by disabling them or marking as insecure. After creation of FFI
+ *        object default rules are added, however caller may add more strict rules or
+ *        completely overwrite rule table by calling rnp_remove_security_rule().
+ *        Note: key signature validation status is cached, so rules should be changed before
+ *        keyrings are loaded or keyring should be reloaded after updating rules.
+ *
+ * @param ffi initialized FFI object.
+ * @param type type of the feature, cannot be NULL. Currently only RNP_FEATURE_HASH_ALG is
+ *             supported.
+ * @param name name of the feature, i.e. SHA1, MD5. The same values are used in
+ *             rnp_supports_feature()/rnp_supported_features().
+ * @param flags additional flags. Following ones currently supported:
+ *              - RNP_SECURITY_OVERRIDE : override all other rules for the specified feature.
+ *                May be used to temporary enable or disable some feature value (like, allow to
+ *                verify SHA1 or MD5 signature), and then revert changes via
+ *                rnp_remove_security_rule().
+ * @param from timestamp, since which rule is active. Objects that have creation time (like
+ *             signatures) are matched with the closest rules from the past, unless there is
+ *             a rule with override flag.
+ * @param level security level of the rule. Currently following ones are defined:
+ *              - RNP_SECURITY_PROHIBITED : feature (for instance, MD5 algorithm) is completely
```

Comment:
I think "for instance" is fine, since the two are interchangeable [1], unless we had agreed to only use one over the other?

[1] https://www.grammarphobia.com/blog/2012/07/for-example-vs-for-instance.html

Reply:
Yeah, `for instance` here seems to be ok.